src/code-i-will-probably-never-use.js

```diff
@@ -0,0 +1,23 @@
+
+
+
+// /\/\/\/\/\//\/\/\/\/\//\/\/\/\/\//\/\/\/\/\//\/\/\/\/\//\/\/\/\/\//\/\/\/\/\//\/\/\/\/\//\/\/\/\/\/
+const parents = document.getElementsByTagName("SELECT");
+const optionsAttr = ["selected", "disabled", "hidden"];
+
+for (let i=0; i<parents.length; i++){
+    
+    var children = parents[i].firstElementChild;
```

Comment:
why childrens implies many first implies just one... there is a little inconsistency here.